File: 1720-crawler-log-folder/crawler-log-folder.dart
class Solution {
  int minOperations(List<String> logs) {
    List<String> stk = [];
    for(String log in logs){
        if (log == './') continue;
        else if (log == '../'){
            if (stk.length > 0){
                stk.removeLast();
            }
        }
        else{
            stk.add(log);
        }
    }
    return stk.length;

    
  }
}

File: 1720-crawler-log-folder/crawler-log-folder.py
class Solution:
    def minOperations(self, logs: List[str]) -> int:
        lgStk = []
        for op in logs:
            if op == './': pass
            elif op == '../':
                if lgStk:
                    lgStk.pop()
            else:
                lgStk.append(op)
        return len(lgStk)
        